Error while checking build/O2/fullCI for 28e7a560679bfbbe648abba8bb631c133aed0dba at 2024-07-24 13:49:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
/sw/SOURCES/O2/13319-slc8_x86-64/0/Detectors/Calibration/workflow/ccdb-populator-workflow.cxx:36:8: error: 'void pol' has incomplete type
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13319/28e7a560679bfbbe648abba8bb631c133aed0dba/build_O2_fullCI/pretty.html).

--- a/Detectors/Calibration/workflow/ccdb-populator-workflow.cxx
+++ b/Detectors/Calibration/workflow/ccdb-populator-workflow.cxx
@@ -33,7 +33,7 @@ void customize(std::vector<o2::framework::CompletionPolicy>& policies)
   // we customize the pipeline processors to consume data as it comes
   using CompletionPolicy = o2::framework::CompletionPolicy;
   using CompletionPolicyHelpers = o2::framework::CompletionPolicyHelpers;
-  auto pol = policies.push_back(CompletionPolicyHelpers::defineByName("ccdb-populator.*", CompletionPolicy::CompletionOp::Consume));
+  auto pol = policies.emplace_back(CompletionPolicyHelpers::defineByName("ccdb-populator.*", CompletionPolicy::CompletionOp::Consume));
   pol.order = CompletionPolicy::CompletionOrder::Slot;
 }
 

--- a/Detectors/Calibration/workflow/ccdb-populator-workflow.cxx
+++ b/Detectors/Calibration/workflow/ccdb-populator-workflow.cxx
@@ -33,7 +33,7 @@ void customize(std::vector<o2::framework::CompletionPolicy>& policies)
   // we customize the pipeline processors to consume data as it comes
   using CompletionPolicy = o2::framework::CompletionPolicy;
   using CompletionPolicyHelpers = o2::framework::CompletionPolicyHelpers;
-  auto pol = policies.emplace_back(CompletionPolicyHelpers::defineByName("ccdb-populator.*", CompletionPolicy::CompletionOp::Consume));
+  auto& pol = policies.emplace_back(CompletionPolicyHelpers::defineByName("ccdb-populator.*", CompletionPolicy::CompletionOp::Consume));
   pol.order = CompletionPolicy::CompletionOrder::Slot;
 }
 

